raiden/messages/healthcheck.py

```diff
@@ -33,7 +33,7 @@ def _data_to_sign(self) -> bytes:
 
 
 @dataclass(repr=False, eq=False)
-class Pong(SignedMessage):
+class Pong(SignedRetriableMessage):
```

Comment:
**Note:** Ping/Pong are currently only used in tests. 

The reason why this change came to be is that previously, the transport relied on `Processed -> Delivered` cycle to test certain aspects of the itself. However, due to the removal of `Delivered` confirmation to `Processed`, the tests became invalid. The way to move forward was to be able to call `send_async` for `Ping` and `Pong` so that these tests could be restored to a passable state. 

The original `MessageHandler` used for Raiden is unable to process `Ping`/`Pong` that's why the `test_matrix_transport.py` `MessageHandler` object was modified to handle `Ping` messages by replying to `Pong`.

I'll wait for the confirmation that the fix works once the build for the current commits has completed.